### DISCUSSION TITLE
Stupid suggestion: mobile push notifications

### DISCUSSION THREAD

**Discussion by The-Abigail608** [ScratchAddons/ScratchAddons#5023] [Ideas] [answered]
ok so my suggestion would be a mobile app that you could use to respond to comments and get push notifications and stuff

**Comment by cobaltt7** [comment, marked as answer]
The Kiwi browser (installable on the Play Store) supports Chrome extensions like SA, and the Scratch Notifier addon works on it. We won't be doing anything for iOS due to difficulty as well as the App Store's $99/year fee.

**Reply by Samq64** [reply]
The notification API works with all platforms including mobile?

**Reply by WorldLanguages** [reply]
Polling the Scratch messages API every minute can drain a phone's battery pretty quickly.
Push notifications exist for a reason. But push is way outside the scope of this project (and if Scratch doesn't support push, someone needs to poll anyway)

**Reply by cobaltt7** [reply]
> The notification API works with all platforms including mobile?

It worked for me
